Automattic/wp-e2e-tests
# Wait for publishSelector, then click on it

In several occasions, we had [this](https://circleci.com/gh/Automattic/wp-e2e-tests/25033#tests/containers/1) failure. I succeeded to reproduce it locally, and it looks like [this](https://github.com/Automattic/wp-e2e-tests/blob/master/lib/gutenberg/gutenberg-editor-component.js#L31) line is passing, but the button is not actually clicked.
Hopefully, wait for this button first will make test more reliable. 

To test:
Make sure that `Calypso Gutenberg Editor: Posts: Insert a payment button` is passing. 
I ran tests several times on CircleCI and didn't see the error again.
